src/core_plugins/kibana/public/management/sections/indices/create_index_pattern_wizard/step_time_field/step_time_field.html

```diff
@@ -0,0 +1,136 @@
+<form
+  role="form"
+  ng-submit="stepTimeField.createIndexPattern()"
+  class="kuiPanel"
+>
+  <div class="kuiPanelHeader">
+    <div class="kuiPanelHeaderSection">
+      <h2 class="kuiPanelHeader__title">
+        Step 2 of 2: Optional settings
+      </h2>
+    </div>
+  </div>
+
+  <div class="kuiPanelBody">
+    <p class="kuiText kuiSubduedText kuiVerticalRhythm">
+      You've defined <strong>{{stepTimeField.indexPatternName}}</strong> as your index pattern. Now you can specify some optional settings before we create it.
+      <small>
+        <a
+          class="kuiLink"
+          ng-click="stepTimeField.toggleAdvancedOptions()"
+        >
```

Comment:
Nice! Good idea about switching between "Show" and "Hide" language. If we're going that far, how about using the ToggleButton component from the UI framework instead of a link?

![image](https://user-images.githubusercontent.com/1238659/29281104-8ff622fc-80d2-11e7-8504-c97bd11717f5.png)